gr-qtgui/CMakeLists.txt

```diff
@@ -29,15 +29,13 @@ include(GrPython)
 set(DESIRED_QT_VERSION 5 CACHE STRING "The Qt version to use (4 or 5)")
```

Comment:
Do we still need this line & the next line? Guessing they can now be removed & the variable replaced by "5" everywhere.

Reply:
I agree, removing that will cleanup the cmake considerably.